index.js

```diff
@@ -8,9 +8,20 @@ const execFile = util.promisify(childProcess.execFile);
 
 const windows = async () => {
 	// Source: https://github.com/MarkTiedemann/fastlist
-	const bin = path.join(__dirname, 'fastlist.exe');
+	let bin;
+	switch (process.arch) {
+		case 'x64':
+			bin = 'fastlist-0.2.1-x64.exe';
```

Comment:
Can you drop the version number from the binary names? We have to remember to update a lot of places when we update it. We can mention the version in a comment here or in a plain `version` file in the same directory.

Reply:
I'm not sure that's a good idea. If we mention the version in a comment or a plain version file, we'd have to update those as well, which is, of course, something that could be forgotten as well... Now, we have to update the switch statement in` index.js` and the files array in the `package.json`. I kind of prefer updating these two locations since it's very explicit and fails fast: If any of them is not updated, the module won't work anymore, which I think is a good thing in this case.

That being said, I'd be fine with dropping the version number from the binary names, too. Let me know. :)

Reply:
My worry is forgetting to update package.json, but I guess that could be solved by moving them into a `vendor` directory.

Reply:
Is the vendor directory uploaded/published automatically? If so, I'd gladly move them there.

Reply:
Confirmed that this would work using `npm publish --dry-run`.